### ISSUE TITLE
What license is this project under?

### ISSUE THREAD

**Issue by zazahunter** [livinamuk/Hell2025#2]
Hi,
You said it is open source but there is no license tied to it.
Without a license we cannot legally use your code in other projects.
MIT means that it is litterly you can take and do what ever you want with it.

**Comment by tengkuizdihar** [comment]
man's living in Tasmania, he's too cool for worldly attachments like "licenses".

**Comment by GhaithA-M** [comment]
yes, the man literally said: "if you believe in time, happy new years"
